packages/playground/src/components/smtp_server.vue

```diff
@@ -20,17 +20,16 @@
       <input-validator
         :value="$props.modelValue.username"
         :rules="[
-          validators.required('Email is required.'),
-          validators.isEmail('Please provide a valid email address.'),
-          v => {
-            return isDiscourse ? undefined : validators.isEmail('Please provide a valid email address.')(v);
-          },
+          validators.required('Email or Username is required.'),
+          (v: string) => {
+            return validators.isValidSmtp(v);
```

Comment:
as we accepts a username now i think we should add min char validation to be at least two chars

Reply:
@AhmedHanafy725 agree?

Reply:
I discussed it with him earlier today, you can go a head :D